packages/cli/src/services/dependencies.ts

```diff
@@ -0,0 +1,25 @@
+import { exec } from 'child-process-promise'
+import { wrapExecError } from '../common/errors'
+import * as path from 'path'
+
+export async function pruneDevDependencies(): Promise<void> {
+  try {
+    await exec('npx yarn install --production --no-bin-links', { cwd: projectDir() })
+  } catch (e) {
+    throw wrapExecError(e, 'Could not prune dev dependencies')
+  }
+}
+
+export async function reinstallDependencies(skipRestoreDependencies: boolean): Promise<void> {
+  try {
+    if (!skipRestoreDependencies) {
+      await exec('npx yarn install', { cwd: projectDir() })
+    }
+  } catch (e) {
+    throw wrapExecError(e, 'Could not reinstall dependencies')
+  }
+}
+
+function projectDir(): string {
+  return path.join(process.cwd())
```

Comment:
I'm not sure if `join` is needed since we are in the user project. Aren't we?

Reply:
Perhaps this is for cross platform compatibility?

Reply:
removed this dumb method in 603b79440028bcf68b3e1ba82b07ec75e3dea108